content/en/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md

```diff
@@ -11,7 +11,7 @@ weight: 10
 
 <!-- overview -->
 
-Kubernetes {{< skew currentVersion >}} supports [Container Network Interface](https://github.com/containernetworking/cni)
+Kubernetes {{< skew currentPatchVersion >}} supports [Container Network Interface](https://github.com/containernetworking/cni)
```

Comment:
The original is fine; this statement applies to all patch versions in a minor release.